doc/manual-installation.md

```diff
@@ -30,6 +30,22 @@ Install the operator.
 $ kubectl create -f deploy/operator.yaml
 ```
 
+#### Deploying in Test Mode
+ We can enable test mode on operator by passing an argument `-test` in `operator.yaml` file. Operator running in test mode skips minimum replica requirement checks on Pravega components.We do not want to encourage users to use "test mode" as it is supposed to be used only in extreme cases where we have severe resource constraints in the environment where Pravega is being deployed.
```

Comment:
"Operator running in test mode skips minimum replica requirement checks on Pravega components.We do not want to encourage users to use "test mode" as it is supposed to be used only in extreme cases where we have severe resource constraints in the environment where Pravega is being deployed."
This has already been mentioned in the Readme. Please provide link to the same instead of repeating it here.

Reply:
Fixed